### ISSUE TITLE
The question for cmumosei.

### ISSUE THREAD

**Issue by AIXiaoBaiDemon** [zinengtang/TVLT#6]
Your work is great, thank you so much for sharing. I downloaded the CMU-MOSEI database according to the link you shared, but I found two problems.
1. there is no labels_emotion folder in the file I downloaded, so I don't know where the emotion label is?
2. There is a 'BA' folder under the 'labels' folder, I don't know what the content stored in it is? The folder doesn't exist in the file directory you're sharing.

**Comment by Changezi001** [comment]
I have the same problem, I am also waiting for the author to share the labels_emotion folder

**Comment by BDHU** [comment]
+1
I suspect the `BA` folder might be for emotion analysis but isn't sure.

**Comment by Changezi001** [comment]
I do not think that the BA folder is the labels_emotion folder.  If it is then what do -1, -2, -3, 0, 1, 2, 3 correspond to? Because the CMU-MOSEI dataset has labels for six basic emotions, why do we have seven here?
@zinengtang could you clear this confusion, please? and could you share the labels_emotion folder? We have been waiting for one month to get this folder.

**Comment by AIXiaoBaiDemon** [comment]
@BDHU @Changezi001 I want to use this method to apply it to real scenes, but almost always it is judged as happy, have you tried this?
